src/components/RegistrationTable/types.ts

```diff
@@ -1,9 +1,12 @@
 import { ColumnDef } from "@tanstack/react-table"
 import { Attendee } from "./columns"
+import { Dispatch, SetStateAction } from "react"
```

Comment:
Do we need this import?

Reply:
I think this is leftover from something I was playing around with before, nice catch